src/OrchardCore/OrchardCore.Abstractions/Modules/Services/ILocalClock.cs

```diff
@@ -5,10 +5,14 @@ namespace OrchardCore.Modules;
 /// </summary>
 public interface ILocalClock
 {
+    [Obsolete("This property has been deprecated, please use GetLocalNowAsync() instead.", error: true)]
```

Comment:
why do we need error: true?

Also 
```suggestion
    [Obsolete("This property is deprecated and will be removed in version 3. Please use GetLocalNowAsync() as the replacement.", error: true)]
```

Reply:
Yeah, `error` shouldn't be there, we just want a warning.